packages/server/auth.config.ts

```diff
@@ -0,0 +1,31 @@
+import { defineConfig } from 'auth-astro';
+
+import { getServerSecrets } from './src/secrets';
+
+const secrets = getServerSecrets((import.meta as any).env);
+
+export default defineConfig({
+  secret: 'secrets.authSecret',
```

Comment:
This shouldn't be hardcoded as a string. I did this to get tests passing in CI, but am looking into using the source environment variable.